api/update.sh

```diff
@@ -0,0 +1 @@
+go test $(go list -f '{{ .ImportPath }} {{ .TestImports }}' ./... | grep gotest.tools/v3/golden | awk '{print $1}' | tr '\n' ' ') -test.update-golden=true
```

Comment:
does this requires to be in separate file?

Reply:
yeah I guess.... we can use it independently as well